buildkite/scripts/debian/install.sh

```diff
@@ -38,6 +38,9 @@ else
         # Downaload mina-logproc too
         ./buildkite/scripts/cache/manager.sh read "debians/$MINA_DEB_CODENAME/mina-logproc*" $LOCAL_DEB_FOLDER
       ;;
+      mina-devnet-legacy|mina-mainnet-legacy)
+        # Download mina-logproc legacy too
+        ./buildkite/scripts/cache/manager.sh read --root "legacy" "debians/$MINA_DEB_CODENAME/${i}*" $LOCAL_DEB_FOLDER
```

Comment:
Why this line is so different from the line above, handling the non-legacy case?

Non-legacy:
```ocaml
./buildkite/scripts/cache/manager.sh read "debians/$MINA_DEB_CODENAME/mina-logproc*" $LOCAL_DEB_FOLDER
```

Legacy:
```ocaml
./buildkite/scripts/cache/manager.sh read --root "legacy" "debians/$MINA_DEB_CODENAME/${i}*" $LOCAL_DEB_FOLDER
```

Reply:
By default root for cache manager is  /var/storagebox/${BUILDKITE_BUILD_ID} ...  above line uses static root. thanks to that, artifact put there won't be cleared by janitor which removed old buildkite_build_id folders.